### DISCUSSION TITLE
What kind of Apple account should I choose?

### DISCUSSION THREAD

**Discussion by andyasberry** [fastlane/fastlane#19329] [Q&A]
Trying to build an app that can be distributed internally to many users (not via App Store). I've seen this done with an Apple Enterprise developer account, but we've had extreme difficulty in setting up a new account for this type.

With respect to Fastlane Match, what is the recommended Apple account type (given the above), and how well will Fastlane Match work with it? 

Trying to learn more, since I've heard there are issues with Fastlane Match and MFA on Apple Enterprise Accounts, etc. I've also read that Apple may be phasing out the Enterprise account, but not sure if that is the case or not.

Many thanks in advance for any advice/information you can provide me with!

**Comment by lucgrabowski** [comment]
I think that the most important thing is what kind of account fits your needs.
If you want to distribute iOS/iPadOS apps to employees within the organization, you could use Enterprise account, but you would have to check eligibility section in https://developer.apple.com/programs/enterprise/

`match` enables you to create certificates and provisioning profiles and it works well with both standard Developer Account and with Enterprise Account https://docs.fastlane.tools/actions/match/

If you use it with Enterprise account, you should read the explanation about security in the documentation:
> Enterprise Profiles
Attackers could use an In-House profile to distribute signed application to a potentially unlimited number of devices. All this would run under your company name and it could eventually lead to Apple revoking your In-House account. However it is very easy to revoke a certificate to remotely break the app on all devices.
Because of the potentially dangerous nature of In-House profiles please use match with enterprise profiles with caution, ensure your git repository is private and use a secure password.

**Comment by Winger96** [comment]
Everything is very open with a clear description of the issues. It was truly informative. Your response is very helpful. Many thanks for sharing!


[www.walgreenslistens.com](https://www.walgreenslistens.kim/)
